esi_leap/objects/contract.py

```diff
@@ -117,7 +117,7 @@ def expire(self, context=None):
 
         resource = o.resource_object()
         if resource.get_contract_uuid() == self.uuid:
-            resource.set_contract(None)
+            resource.expire_contract(self)
```

Comment:
Oh, I forgot - I think the 'cancel' method here also calls set_contract(None); can you update that to call expire_contract as well?

Reply:
Fixed it. Is there any command line corresponding with canceling a contract? I only see 'lease contract delete'

Reply:
It's the delete call; we may have to rethink some of these CLI names at some point.